wgs/workflows/postprocessing/__init__.py

```diff
@@ -32,7 +47,20 @@ def get_coverage_data(input_bam, output, refdir, single_node=False):
             obj=mgd.OutputChunks('chromosome'),
             value=chromosomes
         )
-
+        workflow.transform(
```

Comment:
do you really need to parallelize and run this 24 times?? you can run the task one time but generate 24 files.

example: https://github.com/shahcompbio/biowrappers/blob/master/biowrappers/pipelines/alignment/__init__.py#L29

Reply:
i mean the files get merged in the end...I had dont it for speed